packages/core/src/common/menu/menu-model-registry.ts

```diff
@@ -214,9 +202,9 @@ export class MenuModelRegistry {
      * @param id technical identifier of the `MenuNode`.
      */
     unregisterMenuNode(id: string): void {
-        const recurse = (root: CompositeMenuNode) => {
+        const recurse = (root: MutableCompoundMenuNode) => {
```

Comment:
You're right that it would be fine to use `CompoundMenuNode` here, but since this is declared locally and isn't API, and `this.root` is guaranteed to be mutable (because it must be the `CompositeMenuNode` class or a derivative), I don't think changing the declaration would actually make a difference?

Reply:
Sure, I don't have a strong opinion on this anyway. It just stuck out to me as an unnecessary changed line.